### PR TITLE
SWI-2566 Added detectLanguage

### PR DESCRIPTION
**Pull request by brianluisgomez**

### CODE REVIEW

**Review by ajrice6713 — APPROVED** [review]

File: lib/bandwidth/voice_lib/bxml/verbs/record.rb [hunk 1]
```diff
@@ -19,6 +19,7 @@ def to_bxml(xml)
          'maxDuration' => max_duration,
          'fileFormat' => file_format,
          'transcribe' => transcribe,
+         'detectLanguage' => detectLanguage,
```
Comment (ckoegel) [inline lib/bandwidth/voice_lib/bxml/verbs/record.rb:22]:
```suggestion
         'detectLanguage' => detect_language,
```
ruby uses snake_case for variable names, the instances referring to the `detectLanguage` variable should be changed to `detect_language`

File: lib/bandwidth/voice_lib/voice/models/transcribe_recording_request.rb [hunk 2]
```diff
@@ -54,6 +59,7 @@ def optionals
         password
         tag
         callback_timeout
+        detectlanguage
```
Comment (ckoegel) [inline lib/bandwidth/voice_lib/voice/models/transcribe_recording_request.rb:62]:
```suggestion
        detect_language

```

File: lib/bandwidth/voice_lib/voice/models/transcribe_recording_request.rb [hunk 3]
```diff
@@ -65,6 +71,7 @@ def nullables
...
```
Comment (ckoegel) [inline lib/bandwidth/voice_lib/voice/models/transcribe_recording_request.rb:74]:
```suggestion
        detect_language

```

**Review by ajrice6713 — APPROVED** [review]

File: lib/bandwidth/voice_lib/voice/models/transcribe_recording_request.rb [hunk 4]
```diff
@@ -54,6 +59,8 @@ def optionals
         password
         tag
         callback_timeout
+        detect_language
+
```
Comment (ckoegel) [inline lib/bandwidth/voice_lib/voice/models/transcribe_recording_request.rb:63]:
```suggestion
```

**Review by ajrice6713 — APPROVED** [review]

**Review by ckoegel — APPROVED** [review]

**Pull request merged**